### PR TITLE
submit org.blackfennec.app

### PR DESCRIPTION
**Pull request by SimonKindhauser**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party. 
- [x] I am an upstream contributor to the project.
- [x] I own the domain used in the application ID.
- [x] Any additional patches or files have been submitted to the upstream projects concerned.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: dependencies.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "pypi-dependencies",
+    "buildsystem": "simple",
+    "build-commands": [],
+    "modules": [
+        {
+            "name": "python3-PyGObject",
```
Comment (hfiguiere) [inline dependencies.json:7]: PyGObject is in the GNOME runtime, isn't it?

File: org.blackfennec.app.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+app-id: org.blackfennec.app
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: run.sh
+tags:
+  - devel
+  - development
+  - nightly
```
Comment (hfiguiere) [inline org.blackfennec.app.yml:9]:
```suggestion
```

File: org.blackfennec.app.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  - name: blackfennec
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/applications/
+      - mkdir -p /app/share/icons/hicolor/128x128/apps/
+      - mkdir -p /app/share/metainfo/
+      - cp -r ./ /app/
+      - cp ./org.blackfennec.app.desktop /app/share/applications/org.blackfennec.app.desktop
+      - cp ./icon.png /app/share/icons/hicolor/128x128/apps/org.blackfennec.app.png
+      - cp ./org.blackfennec.app.metainfo.xml /app/share/metainfo/org.blackfennec.app.metainfo.xml
```
Comment (hfiguiere) [inline org.blackfennec.app.yml:24]:
```suggestion
      - cp -r ./ /app/
      - install -Dm644 ./org.blackfennec.app.desktop /app/share/applications/org.blackfennec.app.desktop
      - install -Dm644 ./icon.png /app/share/icons/hicolor/128x128/apps/org.blackfennec.app.png
      - install -Dm644 ./org.blackfennec.app.metainfo.xml /app/share/metainfo/org.blackfennec.app.metainfo.xml
```

**Comment by hfiguiere** [comment]
bot, build org.blackfennec.app

File: dependencies.json (hunk 1 above)
Comment (SimonKindhauser) [reply dependencies.json]: removed dependencies and it still builds... you're right :)

**Review by hfiguiere — CHANGES_REQUESTED** [review]
still some changes not done albeit marked as resolved.

File: dependencies.json (hunk 1 above)
Comment (SimonKindhauser) [reply dependencies.json]: yes ^^

File: org.blackfennec.app.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: run.sh
+tags:
+  - devel
+  - development
+  - nightly
+finish-args:
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
```
[the remark this answers is not in the document]
Comment (SimonKindhauser) [reply org.blackfennec.app.yml:7]: whops;
done;

**Comment by SimonKindhauser** [comment]
should be resolved now

**Comment by SimonKindhauser** [comment]
bot, build org.blackfennec.app

**Pull request closed without merging**